docs/spec/spec.md

```diff
@@ -130,12 +146,22 @@ The database schema will contain the code to create,
 1. Tables for each entity with defined primary keys
 2. Create foreign key associations between tables if the model has defined associations between entities
 
+Apart from the `bal build` users can also use the following command to generate the same as above mentioned,
+
+```bash
+bal persist generate --datastore mysql --module db
+```
+
+| Command Parameter |                                       Description                                        | Mandatory | Default Value  |
+|:-----------------:|:----------------------------------------------------------------------------------------:|:---------:|:--------------:|
+|    --datastore    |  used to indicate the preferred database client. Currently, 'mysql', 'mssql', 'google sheets' and 'postgresql' are supported.   |    Yes     |        |
+|     --module      |      used to indicate the persist enabled module in which the files are generated.       |    No     | <package_name> |
+
 Behaviour of the `generate` command,
 - User should invoke the command within a Ballerina project
-- The user should have initiated the persistence layer in the project and update the model definition file.
 - The model definition file should contain the `persist` module import (`import ballerina/persist as _;`)
 - The Model definition file should contain at least one entity
-- If the user invokes the command twice, it will not fail. It will generate the files once again.
+- If the user invokes the command twice, it will fail. The user needs to remove the persist configurations from the Ballerina.toml if they want to reinitialize the project.
```

Comment:
why can't we execute the generate command twice? which configuration is it?

Reply:
We can execute the generate command twice. Fixed in [5e25761](https://github.com/ballerina-platform/persist-tools/pull/331/commits/5e25761a1d5f197e29a863683df4b4ff22918950)